cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -627,6 +627,13 @@ get_architecture() {
         fi
     fi
 
+    if [ "$_ostype" = Darwin ] && [ "$_cputype" = x86_64 ]; then
+        # Rosetta on aarch64
+        if [ "$(sysctl -n hw.optional.arm64 2>/dev/null)" = "1" ]; then
```

Comment:
I used `sysctl -n` instead of piping into grep for a bit of a cleaner comparison.